src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -2447,19 +2447,17 @@ bool AdaptDispatch::DoConEmuAction(const std::wstring_view string)
     {
         if (parts.size() >= 2)
         {
-            const auto path = til::at(parts, 1);
+            auto path = til::at(parts, 1);
             // The path should be surrounded with '"' according to the documentation of ConEmu.
             // An example: 9;"D:/"
+            // If we fail to find the surrounding quotation marks, we'll give the path a try anyway.
+            // ConEmu also does this.
             if (path.at(0) == L'"' && path.at(path.size() - 1) == L'"' && path.size() >= 3)
```

Comment:
Why is the `path.size()` check not the first thing that's tested? 🤔

Reply:
LOL GOOD QUESTION